Makefile

```diff
@@ -40,7 +41,14 @@ ${API_DIR}/client/${V}/%/http.go: ${PROTO_DIR}/%_openapi.yaml
 		-o ${API_DIR}/client/${V}/$(*F)/http.go \
 		${PROTO_DIR}/$(*F)_openapi.yaml
 
-generate: ${GEN_DIR}/api.pb.go ${GEN_DIR}/api.pb.gw.go ${GEN_DIR}/health.pb.go ${GEN_DIR}/health.pb.gw.go
+${DATA_PROTO_DIR}/%.pb.go ${DATA_PROTO_DIR}/%.pb.gw.go: ${DATA_PROTO_DIR}/%.proto
+	protoc -Iinternal \
+		--go_out=${DATA_PROTO_DIR} --go_opt=paths=source_relative \
+		--go-grpc_out=${DATA_PROTO_DIR} --go-grpc_opt=paths=source_relative \
```

Comment:
We don't need go-grpc and grpc-gateway for now

Reply:
good point